ganga/GangaCore/Utility/feedback_report.py

```diff
@@ -72,57 +71,47 @@ def get_content_type(filename):
         
         return data, file_dict
 
-    def make_upload_file(server):
-
-        def upload_file(path):
-
-            # print 'Uploading %r to %r' % (path, server)
-
-            data = {'MAX_FILE_SIZE': '3145728',
-                    'sub': '',
-                    'mode': 'regist'}
-            files = {'file': path}
-
-            return send_post(server, files)
-
-        return upload_file
+    def upload(filetype, filename, localdir):
+        try:
+                from GangaCore.GPIDev.Adapters.IGangaFile import IGangaFile
+                from GangaCore.GPIDev.Lib.File.LocalFile import LocalFile
+                from GangaCore.GPIDev.Lib.File.LCGSEFile import LCGSEFile
+                from GangaCore.GPIDev.Lib.File.GoogleFile import GoogleFile
+                from GangaCore.GPIDev.Lib.File.MassStorageFile import MassStorageFile
+                
+                filetype = stripProxy(filetype)
+                assert(issubclass(filetype, IGangaFile))
+                print(filename, localdir)
```

Comment:
We should only use the logger to print anything.

Reply:
This was for debugging, will be removed in the next commit